parts/kubernetesagentvars.t

```diff
@@ -1,7 +1,9 @@
     "{{.Name}}StorageAccountOffset": "[mul(variables('maxStorageAccountsPerAgent'),variables('{{.Name}}Index'))]",
     "{{.Name}}Count": "[parameters('{{.Name}}Count')]",
+    "{{.Name}}Offset": "[parameters('{{.Name}}Offset')]",
+    "{{.Name}}TotalCount": "[add(parameters('{{.Name}}Count'), parameters('{{.Name}}Offset'))]",
```

Comment:
Shouldn't count be the goal state. E.g. If current VM count is 4, and desired count is 5  (goal state). So total count will be: 5+4 =9?